lib/collections/initiatives.js

```diff
@@ -64,9 +64,15 @@ Initiatives.before.insert(function initiativesBeforeInsert(userId, doc) {
   doc.createdAt = new Date().getTime();
 });
 
+Initiatives.before.update(function initiativesBeforeUpdate(userId, doc, fieldNames, modifier){
+    if(!_.isEmpty(modifier.$set)){
```

Comment:
Will fail eslint. requires a space after `if` and a space before `{`.

I would also invert this

```
if (_.isEmpty(modifier.$set)) {
  return;
}
modifier.$set.modifiedAt = Date.now();
modifier.$set.categorySlug = s.slugify(modifier.$set.category);
```